### PR TITLE
make the network handler for fabric generic

### PR DESCRIPTION
**Pull request by adecaro**
This allows the developer to plugin different ways to manage the fabric namespaces.

### CODE REVIEW

File: integration/nwo/token/fabric/cc/tcc.go [hunk 1]
```diff
@@ -17,20 +17,139 @@ import (
...
+	var cc *topology.ChannelChaincode
+	for _, chaincode := range p.Fabric(tms).Topology().Chaincodes {
+		if chaincode.Chaincode.Name == tms.Namespace {
+			cc = chaincode
+			break
+		}
+	}
+	Expect(cc).NotTo(BeNil(), "failed to find chaincode [%s]", tms.Namespace)
+
+	packageDir := filepath.Join(
```
Comment (alexandrosfilios) [inline integration/nwo/token/fabric/cc/tcc.go:114]: Maybe we can extract 114-145 to a method

```
func (p *GenericBackend) packageChaincode(tmsID TMSID, cc *topology.ChannelChaincode, chaincodeNameSuffix string) (filename, error)
```

File: integration/nwo/token/fabric/cc/tcc.go [hunk 2]
```diff
@@ -17,20 +17,139 @@ import (
...
+		DefaultTokenChaincodeParamsReplaceSuffix,
+		tokenPlatform,
+	)
+}
+
+func NewGenericBackend(tokenChaincodePath string, tokenChaincodeParamsReplaceSuffix string, tokenPlatform common2.TokenPlatform) *GenericBackend {
+	return &GenericBackend{TokenChaincodePath: tokenChaincodePath, TokenChaincodeParamsReplaceSuffix: tokenChaincodeParamsReplaceSuffix, TokenPlatform: tokenPlatform}
+}
+
+func (p *GenericBackend) PrepareNamespace(entry *fabric.Entry, tms *topology3.TMS) {
```
Comment (alexandrosfilios) [inline integration/nwo/token/fabric/cc/tcc.go:64]: Do we need the entry as a parameter? If another backend comes that needs the entry as input, we can add it later to the interface.

**Review by alexandrosfilios — COMMENTED** [review]
Some comments to address

File: integration/nwo/token/fabric/cc/tcc.go (hunk 2 above)
Comment (adecaro) [reply integration/nwo/token/fabric/cc/tcc.go:64]: okay

File: integration/nwo/token/fabric/cc/tcc.go (hunk 1 above)
Comment (adecaro) [reply integration/nwo/token/fabric/cc/tcc.go:114]: let's address this later

**Pull request merged**